images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -87,19 +96,40 @@ function Install-JavaFromAdoptOpenJDK {
     New-Item -ItemType File -Path $javaVersionPath -Name "$Architecture.complete" | Out-Null
 }
 
-$jdkVersions = (Get-ToolsetContent).java.versions
+$jdkVendors = (Get-ToolsetContent).java.vendors
+$defaultVendor = (Get-ToolsetContent).java.default_vendor
 $defaultVersion = (Get-ToolsetContent).java.default
 
-foreach ($jdkVersion in $jdkVersions) {
-    Install-JavaFromAdoptOpenJDK -JDKVersion $jdkVersion
+foreach ($jdkVendor in $jdkVendors) {
+    $jdkVendorName = $jdkVendor.name
+    $jdkVersionsToInstall = $jdkVendor.versions
+    
+    $isDefaultVendor = $jdkVendorName -eq $defaultVendor
 
-    if ($jdkVersion -eq $defaultVersion) {
-        Set-JavaPath -Version $jdkVersion -Default
-    } else {
-        Set-JavaPath -Version $jdkVersion
+    foreach ($jdkVersionToInstall in $jdkVersionsToInstall) {
+        $isDefaultVersion = $jdkVersionToInstall -eq $defaultVersion
+
+        Install-JavaJDK -VendorName $jdkVendorName -JDKVersion $jdkVersionToInstall
+
+        if ($isDefaultVendor) {
+            if ($isDefaultVersion) {
+            Set-JavaPath -Version $jdkVersionToInstall -VendorName $jdkVendorName -Default
+
+            } else {
+            # Set default vendor and non-default vendor's version
+            # not to coflict with Adopt
```

Comment:
This comment sounds a bit confusing to me. Maybe remove it as the logic is quite clear here?